build.gradle

```diff
@@ -91,6 +95,7 @@ dependencies {
     compile "inventory-tweaks:InventoryTweaks:${config.invtweak_version}:api" // Not updated yet
     compile "li.cil.oc:OpenComputers:MC1.11.2-${config.opencomputers_version}:api" // Not updated yet
     deobfCompile "mcp.mobius.waila:Hwyla:${config.hwyla_version}"
+    deobfCompile "cofh:RedstoneFlux:${config.minecraft_version}-${config.redstoneflux_version}:universal"
```

Comment:
I think you can use plain `compile` here can't you? Their API shouldn't need to be deobfed